src/connector/src/sink/catalog/desc.rs

```diff
@@ -136,3 +136,9 @@ impl SinkDesc {
         }
     }
 }
+
+impl std::hash::Hash for SinkDesc {
+    fn hash<H: std::hash::Hasher>(&self, state: &mut H) {
+        self.id.hash(state);
+    }
+}
```

Comment:
PTAL @chenzl25

Reply:
Besides we can just use id as hashcode, any other motivation to replace BtreeMap with HashMap?

Reply:
we remove the use of Btreemap, and switch to HashMap to avoid unnecessary conversion. Use id as hashcode is to fulfill the `Hash` required by optimizer